src/main/java/com/example/Acmeplex/repositiories/ShowRepository.java

```diff
@@ -0,0 +1,46 @@
+package com.example.Acmeplex.repositiories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+import com.example.Acmeplex.entities.Show;
+
+import java.sql.Date;
+import java.sql.Time;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+
+public interface ShowRepository extends JpaRepository<Show, Integer> {
+
```

Comment:
we have two repositories folder, one is 'repositiories', the other is 'repositories', we need to resolve this